homeassistant/components/zwave_js/repairs.py

```diff
@@ -14,23 +13,37 @@
 class DeviceConfigFileChangedFlow(RepairsFlow):
     """Handler for an issue fixing flow."""
 
-    def __init__(self, node: Node, device_name: str) -> None:
+    def __init__(self, data: dict[str, str]) -> None:
         """Initialize."""
-        self.node = node
-        self.device_name = device_name
+        self.device_name: str = data["device_name"]
+        self.device_id: str = data["device_id"]
 
     async def async_step_init(
         self, user_input: dict[str, str] | None = None
     ) -> data_entry_flow.FlowResult:
         """Handle the first step of a fix flow."""
+        try:
+            async_get_node_from_device_id(self.hass, self.device_id)
```

Comment:
It's enough to do this in the confirm step, right?

Reply:
Yes, the only reason to do it here is to save the user a click. Remove?

Reply:
The user will click to start the flow anyway, right? So we won't be able to completely save the user from a bad flow, right?

Reply:
The repair will show in the list regardless, the only difference is as soon as you click fix, it will abort. But do you have to click into the issue to see the fix button? If that's the case then you are right, user has to click in anyway so one extra click doesn't matter all that much